language-design.md

```diff
@@ -38,18 +160,89 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 
 **Response**
 
-
+Our main takeaway from _Growing a Language_ was that a good language should
+provide users with a small and useful set of building blocks.  From there, the
+programmer should be able to compose the available tools to suit their needs. We
+tried to use this mentality when adapting the Sound library to a more usable
+API.  First, we created a "Sound" class which contained all of the methods for
+editing sounds.  We tried to be consistent--each method took in a Sound (self)
+along with other parameters and returned a new Sound.  This way, the user could
```

Comment:
Was there a reason you chose immutability? I.e. why not stack all of the changes in a single `Sound` instance?